package.json

```diff
@@ -17,6 +17,7 @@
     ]
   },
   "scripts": {
+    "start": "web-dev-server --node-resolve --open /dev/button.html",
```

Comment:
Please keep this section sorted alphabetically.

Reply:
Done.